spec/rubocop/cop/style/method_missing_spec.rb

```diff
@@ -0,0 +1,17 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::MethodMissing do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense for method_missing' do
+    inspect_source(cop, ['class Test',
+                         '  def method_missing; end',
+                         'end'])
+
+    expect(cop.messages).to eq(['Avoid using `method_missing`.'])
+    expect(cop.offenses.empty?).to eq(false)
```

Comment:
It's actually the other way around. We expect `offenses` to not be empty. But IMO checking `messages` is enough since it's a mapping of `offenses` and will always have the same length.